exercises/05/after/Messages/OrderBilled.cs

```diff
@@ -1,9 +1,8 @@
 ﻿using NServiceBus;
 
-namespace Messages
+namespace Messages.Events
```

Comment:
Was this intentional?

Reply:
It was, because the endpoint is configured with unobtrusive mode based on the namespace